Find a functionally equivalent code:android.text.format.Time.parse3339:COMMENT
Method Modifier: public      
Comment:/**
 * Parse a time in RFC 3339 format.  This method also parses simple dates
 * (that is, strings that contain no time or time offset).  For example,
 * all of the following strings are valid:
 *
 * <ul>
 * <li>"2008-10-13T16:00:00.000Z"</li>
 * <li>"2008-10-13T16:00:00.000+07:00"</li>
 * <li>"2008-10-13T16:00:00.000-07:00"</li>
 * <li>"2008-10-13"</li>
 * </ul>
 *
 * <p>
 * If the string contains a time and time offset, then the time offset will
 * be used to convert the time value to UTC.
 * </p>
 *
 * <p>
 * If the given string contains just a date (with no time field), then
 * the {@link #allDay} field is set to true and the {@link #hour},
 * {@link #minute}, and  {@link #second} fields are set to zero.
 * </p>
 *
 * <p>
 * Returns true if the resulting time value is in UTC time.
 * </p>
 *
 * @param s the string to parse
 * @return true if the resulting time value is in UTC time
 * @throws android.util.TimeFormatException if s cannot be parsed.
 */

@@ -2,8 +2,8 @@
     if (s == null) {
         throw new NullPointerException("time string is null");
     }
-    if (nativeParse3339(s)) {
+    if (parse3339Internal(s)) {
         timezone = TIMEZONE_UTC;
         return true;
     }

